utils/remote_control/navigator_joystick_control/nodes/navigator_joystick.py

```diff
@@ -1,62 +1,31 @@
 #!/usr/bin/env python
 
 
-import itertools
-
-from geometry_msgs.msg import WrenchStamped
-from kill_handling.broadcaster import KillBroadcaster
-from nav_msgs.msg import Odometry
-from navigator_alarm import AlarmBroadcaster
-from navigator_msgs.srv import WrenchSelect, ShooterManual, ShooterManualRequest
-from navigator_msgs.msg import ShooterDoAction, ShooterDoActionGoal
+from remote_control_lib import RemoteControl
 import rospy
-import actionlib
 from sensor_msgs.msg import Joy
-from std_srvs.srv import SetBool, SetBoolRequest, Trigger, TriggerRequest
 
-rospy.init_node("joystick")
+
+__author__ = "Anthony Olive"
```

Comment:
I've rewritten nearly every line in this file by now, but I'll remove it if you like. It's mainly there so that everyone knows who to yell at when things break.